### ISSUE TITLE
Bug of delimiter-dangle for inexact object types?

### ISSUE THREAD

**Issue by apsavin** [gajus/eslint-plugin-flowtype#445]
.eslintrc.js:
```
'flowtype/delimiter-dangle': [2, 'always-multiline'],
```
code:
```
type A = {
  a: string,
  ...
}
```

Expected result: no errors
Actual result: error
Version: `"eslint-plugin-flowtype": "^4.7.0",`

**Comment by gajus** [comment]
Cannot replicate. Please PR a test case.
